scripts/firedrake-install

```diff
@@ -1416,8 +1416,19 @@ def build_and_install_slepc():
 
 
 def build_and_install_netgen():
-    log.info("Installing NGSolve/Netgen.")
-    run_pip(["install", "ngsolve", "--pre"])
+    log.info("Installing scikit-build.")
+    run_pip(["install", "-U", "scikit-build"])
+    log.info("Installing Netgen, it may take an hour or more to build!")
```

Comment:
This seems bad.

Reply:
I think more because building Netgen for an _hour_ is going to significantly slow down CI

Reply:
What do mean you need a branch? I'm not familiar with wheels I'll start looking into it :)

Reply:
Line 1422 appears to indicate that you're doing this because you need your fork of netgen rather than the version on PyPI. 

The version on PyPI will be compiled python packages, which are called wheels. If it takes this long to build netgen then you need to compile your own Python packages and use them. Netgen is already set up as a Python package, so building the wheels should be easy (see https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives)

You can either put these packages on PyPI under an alternative name, or you can make a release of your project on GitHub and grab the wheels from there. You can look at how we deal with vtk in the install script as an example.